backend/python/app/graphql/mutations/story_mutation.py

```diff
@@ -48,6 +48,25 @@ def mutate(root, info, story_translation_data):
             raise Exception(error_message if error_message else str(e))
 
 
+class CreateStoryTranslationTest(graphene.Mutation):
+    class Arguments:
+        user_id = graphene.ID(required=True)
+        level = graphene.Int(required=True)
+        language = graphene.String(required=True)
+
+    story = graphene.Field(lambda: CreateStoryTranslationResponseDTO)
```

Comment:
nit: would it make more sense for this to be called `storyTranslation` instead?

Reply:
good point - I'll leave it as story for now since the frontend is expecting "story" in the gql request return